Assets/MRTK/SDK/Experimental/InteractiveElement+StateVisualizer/InteractiveElement/Events/EventConfigurations/BaseInteractionEventConfiguration.cs

```diff
@@ -7,7 +7,7 @@
 using System.Resources;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.UI.Interaction
+namespace Microsoft.MixedReality.Toolkit.Experimental.Interaction
```

Comment:
Same here, recommend moving InteractiveElement to experimental in a separate PR to keep this a bit simpler.